src/mod/codecs/mod_amrwb/mod_amrwb.c

```diff
@@ -587,18 +772,37 @@ SWITCH_MODULE_LOAD_FUNCTION(mod_amrwb_load)
 				if (!strcasecmp(var, "force-oa")) {
 					globals.force_oa = (switch_byte_t) atoi(val);
 				}
+				if (!strcasecmp(var, "force-be")) {
+					globals.force_be = (switch_byte_t) atoi(val);
+				}
 				if (!strcasecmp(var, "mode-set-overwrite")) {
 					globals.mode_set_overwrite = (switch_byte_t) atoi(val);
 				}
+				if (!strcasecmp(var, "mode-set-overwrite-with-default-bitrate")) {
+					globals.mode_set_overwrite_with_default_bitrate = (switch_byte_t) atoi(val);
+				}
+				if (!strcasecmp(var, "invite-prefer-oa")) {
+					globals.invite_prefer_oa = (switch_byte_t) atoi(val);
```

Comment:
Should we have a default preference if invite_prefer_oa or invite_prefer_be is not set? Otherwise this [line](https://github.com/team-telnyx/freeswitch/pull/167/files#diff-70ccf3d2216fcb3f902808582faa6d967dd7dfe8f0f41f3cacd985cb9982b9e0R261) will always return false

Reply:
Thanks @dev-ryanc These are set to 0 by default, that means feature is not used (you got no preference). 

I added these params as a way of expressing that you have a preference, and if you don't (both `invite_prefer_oa` or `invite_prefer_be` are not set) then this means you do not prefer either of them - in this case codec selection code will work in default mode, the whole prefer thing is not used. This means (a default) that first AMR-WB from SDP offer will be selected.
In other words we can say that default is select first that is listed on SDP (when both `invite_prefer_oa` or `invite_prefer_be` are not set).

Setting `invite_prefer_oa` or `invite_prefer_be` to 0 is same as not setting them at all (these are flippers to turn feature on/off).

Reply:
Got it